alpha-lab/phone-neon/index.md

```diff
@@ -1,118 +1,103 @@
-while gazing at small icons on mobile applications. To achieve this, we used some of Alpha Lab's existing tutorials, including
-how to generate scanpaths, define areas of interest (AOIs) and calculate outcome metrics, and map gaze onto dynamic content.
+## Gaze Behaviour During Mobile App Engagement
 
-Shall we start?
+Researchers in fields such as UI/UX and neuromarketing frequently investigate user visual engagement with mobile applications. Wearable eye tracking is sometimes used for this purpose as it can provide valuable insights into user experience during mobile browsing. In this article, we used our latest wearable eye tracker, Neon, to capture and analyse users' viewing behaviour when focusing on small icons and features of mobile applications.
 
-## What you'll need
+## Overcoming Usability Challenges With Neon
 
-Below you can find the tools we used for this project. Using these, you can replicate the content of this article with
-your own applications.
+In the past, wearable eye tracking faced challenges in usability primarily due to calibration limitations, which may have made mobile browsing measurements difficult and/or time-consuming in practice. However, we believe that Neon overcomes these obstacles by offering calibration-free gaze estimation with very good accuracy. For this reason, we wanted to try out Neon when tracking gaze during mobile browsing, and put together an article so you can replicate what we did.
 
-### Cloud enrichment
+## Leveraging Pupil Cloud and Alpha Lab Tutorials To Explore User Gaze Dynamics
 
-- [Reference Image Mapper](https://docs.pupil-labs.com/neon/pupil-cloud/enrichments/reference-image-mapper/)
+In order to characterise viewing behaviour on mobile phones using Neon, we first needed to make some eye tracking recordings whilst a user was engaging with a phone. We then leveraged Pupil Cloud and Alpha Lab tutorials to interpret the data and compute further metrics.
 
-### Alpha Lab tutorials
+Firstly, we placed the phone in a static position on a desk, allowing us to interact with the app.
 
-- [How to generate scanpaths](/scanpath-rim/)
-- [How to define areas of interest (AOIs) and calculate basic metrics](/gaze-metrics-in-aois/)
-- [How to map and visualise gaze onto dynamic screen content](/map-your-gaze-to-a-2d-screen/)
+Then, we used Pupil Cloud's [Reference Image Mapper](https://docs.pupil-labs.com/neon/pupil-cloud/enrichments/reference-image-mapper/) to map gaze onto a 2D reference image of the environment that contained our test phone, like in the videos and snapshots.
 
-### How we used them
+Subsequently, we exported the data to analyse offline for a more in-depth exploration. For the offline analysis, we used existing Alpha Lab tutorials that allowed us to [generate scanpaths](/scanpath-rim/), [define areas of interest (AOIs) and calculate gaze metrics](/gaze-metrics-in-aois/), and [map gaze onto dynamic content](/map-your-gaze-to-a-2d-screen/).
```

Comment:
Can we break this into 1, 2, 3 style steps? Instead of First, Then, Subsequently?